input/metadata.json

```
@@ -0,0 +1,989 @@
+              "description": "Average Mature Body Weight (kg/head) -- The average mature body weight of cows (default is 1200)",
+              "default": 1200,
+              "minimum": 0
+            },
+            "mature_body_weight_std": {
+              "type": "number",
+              "description": "Mature Body Weight Standard Deviation (kg/head) -- The standard deviation of mature body weight of cows (default is 73.5)",
+              "default": 73.5,
+              "minimum": 0
+            }
+          }
+        },
+        "from_literature": {
+          "type": "object",
+          "description": "From Literature",
+          "repro": {
+            "type": "object",
+            "description": "Literature Repro Values",
+            "preg_check_day_1": {
+              "type": "number",
+              "description": "Pregnancy Check Day 1 (days) -- Days Since Last Heat of pregnancy check 1 (default is 32)",
+              "default": 32,
+              "minimum": 0
+            },
+            "preg_loss_rate_1": {
+              "type": "number",
+              "description": "Pregnancy Loss Rate 1 -- The % of cows that abort their pregnancy before the 1st pregnancy check (default is 0.02)",
+              "default": 0.02,
+              "minimum": 0,
+              "maximum": 0.1
+            },
+            "preg_check_day_2": {
+              "type": "number",
+              "description": "Pregnancy Check Day 2 (days) -- Days Since Last Heat of pregnancy check 2 (default is 60)",
+              "default": 60,
+              "minimum": 0
+            },
+            "preg_loss_rate_2": {
+              "type": "number",
+              "description": "Pregnancy Loss Rate 2 -- % of cows that abort their pregnancy between the 1st and 2nd pregnancy check (default is 0.096)",
+              "default": 0.096,
+              "minimum": 0,
+              "maximum": 0.1
+            },
+            "preg_check_day_3": {
+              "type": "number",
+              "description": "Pregnancy Check Day 3 (days) -- DCC of pregnancy check 3 (default is 172)",
+              "default": 172,
+              "minimum": 0
+            },
+            "preg_loss_rate_3": {
+              "type": "number",
+              "description": "Pregnancy Loss Rate 3 -- The % of cows that abort their pregnancy between the 2nd and 3rd pregnancy check (default is 0.017)",
+              "default": 0.017,
+              "minimum": 0,
+              "maximum": 0.1
+            }
+          }
+        }
+      },
+      "methane_mitigation": {
+        "type": "object",
+        "description": "Methane Mitigation",
+        "methane_mitigation_method": {
+          "type": "string",
+          "description": "Methane Mitigation Method -- The type of enteric methane mitigation supplement that is fed to the lactating dairy cattle",
+          "default": "None",
+          "pattern": "^{None|3-NOP|monensin|essential_oils|seaweed}$"
```

I'm not sure if json.load will correctly translate "None" from a json file. I think the way None is used in the regex pattern would still work but I will think about that a little longer as well.

```suggestion
          "default": null,
          "pattern": "^{None|3-NOP|monensin|essential_oils|seaweed}$"
```